_posts/2024-02-20-moving-to-zulip.markdown

```diff
@@ -0,0 +1,105 @@
+community and offers several advantages for our members.
+
+Join here → <https://ossci.zulipchat.com/>
+
+### Why Zulip? 🤔
+
+Zulip offers a robust and versatile platform for communication and
+collaboration. Its threading model allows for organized and focused
+discussions, making it easier for community members to follow and participate
+in conversations effectively. Additionally, Zulip provides powerful search
+capabilities, ensuring that valuable information shared in the past remains
+accessible to all.
+
+Zulip's unique message sharing feature allows conversations to be easily shared
+around the web via unique links. In addition, Zulip's indexing of all content
+by search engines ensures that the knowledge base is easily accessible to all
+users.
+
+### Hosting Thanks 🙏🏻
+
+We extend our sincere gratitude to the good humans at the [Open Source Science
+Initiative](https://opensource.science/) (OSSCi) for generously hosting the
+Zulip server. Their commitment to supporting open science and collaborative
+research is commendable, and we're thrilled to partner with them on this
+endeavour. 
+
+Shoutout to [Jonathan Starr](https://www.linkedin.com/in/jonathan-starr-b04032284/)
+for helping us! 🙌🏻
+
+### Building a Hub for Open Science 🧬
+
+The OSSCi Zulip server will serve as a hub for various projects in the
+scientific Python ecosystem, starting with Zarr. By centralising communication
+within this platform, we aim to foster greater collaboration, knowledge
+sharing, and community building among like-minded individuals passionate about
+open science and research.
+
+### Official Chat Platform ™️
+
+With this migration, the OSSCi Zulip server becomes the official chat platform
+for the Zarr community. We encourage all Zarr users, contributors, and
+enthusiasts to join us on Zulip to stay updated on the latest developments,
+seek assistance, and engage with fellow community members.
+
+### Your Feedback Matters 🔁
+
+At Zarr, we value the input and ideas of our community members. We're committed
+to continuously improving our platform and user experience. Therefore, we
+welcome any feedback, suggestions, or ideas you may have regarding the Zulip
+migration or any other aspect of our community. Your input helps us better
+serve the needs of our users and advance our shared goals.
+
+Please create an issue in [zarr-developers/community](https://github.com/zarr-developers/community/issues)
+or join one of our [community meetings](https://zarr.dev/community-calls/)
+if you'd like to chat with us! 
+
+### Join Us on Zulip! 🔗
+
+Ready to join the conversation? Head over to the [OSSCi Zulip](https://ossci.zulipchat.com/)
+server and dive into discussions surrounding Zarr and other exciting projects 
+in the scientific Python ecosystem. We look forward to connecting with you
+there!
+
+> Transition Note: Initial Member Count 📝
+
+Please note that during our transition from Gitter to Zulip, new members joining
```

Comment:
Do we need to talk about the member count? I feel like we shouldn't put up any kind of disclaimer that might remotely discourage people from joining, as this only draws attention to something many people might otherwise ignore. Also, if zulip _does_ see an increase in membership (which is the goal of this post), then this statement might not even be true / applicable (because new members would see that zulip would has a lot of people in it)

Reply:
maybe another way to subtly convey the same sentiment / temper people's expectations is to say "as of this writing, most of the conversations in Zulip so far have been between core Zarr developers, but we want to emphasize that _anyone_ is welcome to join, even if you have never used Zarr!"

Reply:
Sounds good!
Thanks for the suggestion, @d-v-b.